integration-tests/src/test/java/com/datastax/dse/driver/api/core/graph/GraphPagingIT.java

```diff
@@ -69,6 +69,9 @@
                   .withStringList(
                       DefaultDriverOption.METRICS_NODE_ENABLED,
                       Collections.singletonList(DseNodeMetric.GRAPH_MESSAGES.getPath()))
+                  // Increase heartbeat interval to avoid interferences since we pause/stop the
+                  // server in this test
+                  .withDuration(DefaultDriverOption.HEARTBEAT_INTERVAL, Duration.ofMinutes(10))
```

Comment:
If we remove the invalid test below, I don't think we need to increase the heartbeat here. The rest of the tests in this class pass fine without even coming close to tripping the 30 second heartbeat timeout.